scania-digital-design-system/sdds-website
# updated search bar colours

**Describe pull-request**  

updated colours on Search bar and search list 
- The containers around the "getting started", "resources" etc, changed from blue-100 to grey-300.
- Text inside the searchbar changed from blue-200 to grey-400

**Solving issue**  
Fixes: [AB#1142](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1142)

**How to test**  
1. start the project 
2. check search bar text colour and containers colour
